src/tests/functional/shared_test_classes/src/subgraph/constant_result.cpp

```diff
@@ -38,35 +93,37 @@ std::string ConstantResultSubgraphTest::getTestCaseName(const testing::TestParam
     return result.str();
 }
 
-void ConstantResultSubgraphTest::createGraph(const ConstantSubgraphType& type, const SizeVector &inputShape, const Precision &inputPrecision) {
+void ConstantResultSubgraphTest::createGraph(const ConstantSubgraphType& type,
+                                             const InferenceEngine::SizeVector& inputShape,
+                                             const InferenceEngine::Precision& inputPrecision) {
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(inputPrecision);
 
-    ParameterVector params;
-    ResultVector results;
+    ov::ParameterVector params;
+    ov::ResultVector results;
     switch (type) {
-        case ConstantSubgraphType::SINGLE_COMPONENT: {
-            auto input = builder::makeConstant<float>(ngPrc, inputShape, {}, true);
-            results.push_back(std::make_shared<opset3::Result>(input));
-            break;
-        }
-        case ConstantSubgraphType::SEVERAL_COMPONENT: {
-            auto input1 = builder::makeConstant<float>(ngPrc, inputShape, {}, true);
-            results.push_back(std::make_shared<opset3::Result>(input1));
-            auto input2 = builder::makeConstant<float>(ngPrc, inputShape, {}, true);
-            results.push_back(std::make_shared<opset3::Result>(input2));
-            break;
-        }
-        default: {
-            throw std::runtime_error("Unsupported constant graph type");
-        }
+    case ConstantSubgraphType::SINGLE_COMPONENT: {
+        auto input = ngraph::builder::makeConstant<float>(ngPrc, inputShape, {}, true);
+        results.push_back(std::make_shared<ov::op::v0::Result>(input));
+        break;
+    }
+    case ConstantSubgraphType::SEVERAL_COMPONENT: {
+        auto input1 = ngraph::builder::makeConstant<float>(ngPrc, inputShape, {}, true);
```

Comment:
we can move constant creation to original OV API to avoid builders :)

Reply:
We have a dedicated activity to revise builders